### DISCUSSION TITLE
Parse Prompt Attention Question

### DISCUSSION THREAD

**Discussion by geocine** [kohya-ss/sd-scripts#94] [Q&A]
I saw this example
```sh
>>> parse_prompt_attention('a (((house:1.3)) [on] a (hill:0.5), sun, (((sky))).')
  [['a ', 1.0],
   ['house', 1.5730000000000004],
   [' ', 1.1],
   ['on', 1.0],
   [' a ', 1.1],
   ['hill', 0.55],
   [', sun, ', 1.1],
   ['sky', 1.4641000000000006],
   ['.', 1.1]]
  ```
  I have a few questions @kohya-ss 
- How did `on` become `1.0`? I was expecting `on` will be `-1.1` 
- How did `a` , `sun`, `.` become `1.1` ? I was expecting these will be `1.0`
- Is the extra `(` on ` (((house:1.3))` intended on this example. Does it affect the next words or it just becomes ` ((house:1.3))` without the extra ` (`?

**Comment by laksjdjf** [comment]
I think the extra `(` has an effect of 1.1x , since `(` simply multiplies the next words by 1.1 regardless of the position of `)`.
`)` cancels the effect of `(` by dividing the next words by 1.1.
